Quaver.Shared/Skinning/SkinKeys.cs

```diff
@@ -299,9 +299,64 @@ internal SkinKeys(SkinStore store, GameMode mode)
             // skin.ini.
             ReadConfig(true);
             ReadConfig(false);
+            FixScale();
             LoadTextures();
         }
 
+        /// <summary>
+        ///     Scale a skin position or size value to match the UI redesign coordinate system.
+        /// </summary>
+        /// <param name="value"></param>
+        /// <returns></returns>
+        private static int Scale(int value) => value * 1920 / 1366;
+        private static byte Scale(byte value) => (byte) (value * 1920 / 1366);
+
+        /// <summary>
+        ///     Scale all necessary skin keys to match the UI redesign coordinate system.
+        /// </summary>
+        private void FixScale()
```

Comment:
Already talked about this, but this manual scaling can/should be solved by using attributes & reflection.

Here's an example:

`ScalableSkinValue.cs`
```cs
public class ScalableSkinValue : System.Attribute
{
}
```

`SkinKeys.cs`

```cs
[ScalableSkinValue]
public int ColumnSize { get; private set; }

[ScalableSkinValue]
public int ReceptorPosOffsetY { get; private set; }
```

Then after loading both default and user config, you can use reflection to set the scaled values:

```cs
public SkinKeys(SkinStore store, GameMode mode)
{
    Store = store;
    Mode = mode;

    ReadConfig(true);
    ReadConfig(false);
    LoadTextures();

    var props = GetType().GetProperties()
                .Where(prop => Attribute.IsDefined(prop, typeof(ScalableSkinValue))).ToList();

    const int scale = 1920 / 1366;

    foreach (var prop in props)
    {
        var value = prop.GetValue(this);
        
        switch (value)
        {
            case int val:
                prop.SetValue(this, val * scale);
                break;
            case float val:
                prop.SetValue(this, val * scale);
                break;
            case byte val:
                prop.SetValue(this, val * scale);
                break;
        }
    }
}
```